"Zoom right in" tooltip for media embeds.

Erquint opened the pull request:
<!-- Put X in the box below to confirm -->

## Checklist:

- [x] I have read the [Contributing guidelines](https://github.com/SillyTavern/SillyTavern/blob/release/CONTRIBUTING.md).

Following up on #3793 with a tooltip change to "Expand and zoom right in" for the magnifying glass button.

I'm open to suggestions. Other options are:
- "Expand&Zoom"
- "Zoom in"
- "Blow up"
- Suggest your own.

Erquint committed:
"Zoom right in" tooltip for media embeds.

Cohee1207 commented on public/index.html:
```diff
@@ -6450,7 +6450,7 @@ <h3 data-i18n="prompt_manager_edit">Edit</h3>
                     <div class="mes_img_container">
                         <div class="mes_img_controls">
-                            <div title="Enlarge" class="right_menu_button fa-lg fa-solid fa-magnifying-glass mes_img_enlarge" data-i18n="[title]Enlarge"></div>
+                            <div title="Expand and zoom right in" class="right_menu_button fa-lg fa-solid fa-magnifying-glass mes_img_enlarge" data-i18n="[title]Enlarge"></div>
```
"Expand and zoom" would be better. "Right in" part is not needed in my opinion. Also don't forget about data-i18n attribute

Erquint replied:
I was trying to move away from the "enlarge" phrasing as it conveys more of a quantitative notion rather than the modal UI action that it represents. I. e. "enlarge" just as well describes zooming in by some arbitrary amount, making "enlarge and zoom" easily interpreted as a duplicitous notion of "make big and also big." I've often seen "expand" associated with modal actions in other software.

I'm fine with whatever you insist on, but suggest you reconsider.

As for i18n — I didn't touch it, since I am assuming the present ID is already referenced as is internally. Also I'm not intimately familiar i18n.

Cohee1207 replied:
I'm more concerned about "right in" part. I've never seen this phrasing being used in UI texts. And the meaning is not lost if it's removed.

Erquint replied:
With "right in" I was conveying the level of zoom bringing logical pixel ratio 1:1 with physical — that seems like what is being performed — to disambiguate the level of zoom. I can see how it may not read Service-Tensor-like enough, however.

```suggestion
                            <div title="Expand and zoom" class="right_menu_button fa-lg fa-solid fa-magnifying-glass mes_img_enlarge" data-i18n="[title]Enlarge"></div>
```

Cohee1207 replied:
But there's no other way you can zoom except 1:1 ("right in"). You don't have to make up a new term that doesn't make any difference.

Cohee1207 replied:
> As for i18n — I didn't touch it, since I am assuming the present ID is already referenced as is internally. Also I'm not intimately familiar i18n.

This still have to be resolved. Otherwise localizations would still use the old text. Placing a new i18n attribute would mean that they have to be retranslated, which makes sense.

Erquint replied:
> no other way you can zoom except 1:1

Depends on if you're thinking from the point of view of a regular or a newbie.

And oh, okay. I understand about i18n now.

Erquint committed:
Let's not say "right in" in the expand&zoom tooltip then.

Maintainer request in PR: https://github.com/SillyTavern/SillyTavern/pull/3804#discussion_r2032999775

Erquint commented on public/index.html:
```diff
@@ -6450,7 +6450,7 @@ <h3 data-i18n="prompt_manager_edit">Edit</h3>
                     <div class="mes_img_container">
                         <div class="mes_img_controls">
-                            <div title="Enlarge" class="right_menu_button fa-lg fa-solid fa-magnifying-glass mes_img_enlarge" data-i18n="[title]Enlarge"></div>
+                            <div title="Expand and zoom" class="right_menu_button fa-lg fa-solid fa-magnifying-glass mes_img_enlarge" data-i18n="[title]Enlarge"></div>
```
```suggestion
                            <div title="Expand and zoom" class="right_menu_button fa-lg fa-solid fa-magnifying-glass mes_img_enlarge" data-i18n="[title]Expand and zoom"></div>
```

Will this do?

Cohee1207 committed:
Update public/index.html

Yeah

Co-authored-by: Murad "Gness Erquint" Beybalaev <gness.na@gmail.com>
